### DISCUSSION TITLE
WE NEED A FIX ASAP!!!

### DISCUSSION THREAD

**Discussion by Alex2Mayhem** [3kh0/ext-remover#121] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Please Give US A FIX for chrome.google.com/webstorex and Ingot. Please get it done quick there are no other OPTIONS!!

**Comment by ghost** [comment, marked as answer]
Thanks for giving us inspiration!! we will make sure to become wizards and somehow make one faster. Again, thanks!

**Reply by Alex2Mayhem** [reply]
Well heres some tips on how to fix it 
![image](https://user-images.githubusercontent.com/104642381/197591874-64df3321-f78f-49e9-844d-452b6fad748e.png)
 Look at the image. Thats from Chrome policies. try to find a way to change the policy or Block the policies out

**Reply by ghost** [reply]
Honey poo poo, you just sent me a bunch of your meeting links. PS: I can't change what your admin does

**Reply by Alex2Mayhem** [reply]
> Honey poo poo, you just sent me a bunch of your meeting links. PS: I can't change what your admin does

:/

**Reply by ghost** [reply]
sorry lul

**Reply by Meta0Music** [reply]
lemme just explain why rq,

not only has your school administrator blocked all javascript functions, you have most likely updated your chromebook. even if you hadn't the only other way around that is to use an html file apparently.

so basically be patient, in the meantime deploy 3xh0.github.io and just play games there. OR use a proxy, i highly recommend nebula since it seems to be one of the proxies on the faster end, but good ol' ultraviolet works just fine (you can get to it though proxies like utopia) 
if you want the proxy links just dm me on discord at Zytrx#8699
